python/uv/__main__.py

```diff
@@ -1,35 +1,17 @@
 import os
 import sys
 import sysconfig
-
-
-def detect_virtualenv() -> str:
-    """
-    Find the virtual environment path for the current Python executable.
-    """
-
-    # If it's already set, then just use it
-    value = os.getenv("VIRTUAL_ENV")
-    if value:
-        return value
-
-    # Otherwise, check if we're in a venv
-    venv_marker = os.path.join(sys.prefix, "pyvenv.cfg")
-
-    if os.path.exists(venv_marker):
-        return sys.prefix
-
-    return ""
+from pathlib import Path
```

Comment:
We are explicitly not using `pathlib` because it's slower to import

https://github.com/astral-sh/ruff/pull/9315

Reply:
Can you revert those changes?